nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -58,20 +58,23 @@ class OVConvolutionMetatype(OVOpMetatype):
     name = 'ConvOp'
     op_names = ['Convolution']
     hw_config_names = [HWConfigOpName.CONVOLUTION]
+    const_channel_axis = [0]  # const layout: [C_OUT, C_IN, Z, Y, X]
```

Comment:
Should we add default `const_channel_axis=0` for `OVOpMetatype`?
ref: https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize.py#L374